# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ardour/no_gtk1_path.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Nur ardour-0.99.original/libs/gtkmm/configure.ac ardour-0.99/libs/gtkmm/configure.ac
--- ardour-0.99.original/libs/gtkmm/configure.ac	2005-03-12 21:22:08.000000000 +0200
+++ ardour-0.99/libs/gtkmm/configure.ac	2005-10-14 22:51:45.000000000 +0300
@@ -169,7 +169,7 @@
 AC_SUBST(GTK_RELEASE)
 
 dnl we now require 1.2 gtk+ - 1.1 is not allowed because it doesnt work.
-AM_PATH_GTK(1.2.7,,exit)
+dnl AM_PATH_GTK(1.2.7,,exit)
 
 dnl these gen_h stuffs are here because we want more than one
 dnl makefile have access to this variable.
